lib/seal.rb

```diff
@@ -33,26 +33,41 @@ def teams
   def bark_at(team)
     message_builder = MessageBuilder.new(pull_requests(team))
     message = message_builder.build
-    slack = SlackPoster.new(ENV['SLACK_WEBHOOK'], team_config(team)['channel'], message_builder.poster_mood)
+    channel = ENV["SLACK_CHANNEL"] ? ENV["SLACK_CHANNEL"] : team_config(team)['channel']
+    slack = SlackPoster.new(ENV['SLACK_WEBHOOK'], channel, message_builder.poster_mood)
     slack.send_request(message)
   end
 
   def org_config
-    @org_config ||= YAML.load_file("./config/#{ORGANISATION}.yml")
+    @org_config ||= YAML.load_file('./config/#{ORGANISATION}.yml') if File.exist?('./config/#{ORGANISATION}.yml')
```

Comment:
The number of conditionals in this class concern me.

It feels like we need `EnvironmentConfiguration` and `FileConfiguration` `class`es with a common `Configuration` interface.